<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="3" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-10-02T21:55:09 IST" name="Suite" finished-at="2022-10-02T21:56:27 IST" duration-ms="77846">
    <groups>
      <group name="ErrorHandling">
        <method signature="EndToEndTest2ErrorCheck.EndToEndtest()[pri:0, instance:parent.testClass.EndToEndTest2ErrorCheck@f58853c]" name="EndToEndtest" class="parent.testClass.EndToEndTest2ErrorCheck"/>
      </group> <!-- ErrorHandling -->
    </groups>
    <test started-at="2022-10-02T21:55:09 IST" name="EtoETest" finished-at="2022-10-02T21:56:08 IST" duration-ms="59339">
      <class name="parent.testClass.EndToEndTest">
        <test-method is-config="true" signature="launchURL()[pri:0, instance:parent.testClass.EndToEndTest@790da477]" started-at="2022-10-02T21:55:09 IST" name="launchURL" finished-at="2022-10-02T21:55:33 IST" duration-ms="23840" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchURL -->
        <test-method signature="EndToEndtest()[pri:0, instance:parent.testClass.EndToEndTest@790da477]" started-at="2022-10-02T21:55:33 IST" name="EndToEndtest" finished-at="2022-10-02T21:56:00 IST" duration-ms="26673" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- EndToEndtest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:parent.testClass.EndToEndTest@790da477]" started-at="2022-10-02T21:56:00 IST" name="tearDown" finished-at="2022-10-02T21:56:00 IST" duration-ms="157" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchURL()[pri:0, instance:parent.testClass.EndToEndTest@790da477]" started-at="2022-10-02T21:56:00 IST" name="launchURL" finished-at="2022-10-02T21:56:07 IST" duration-ms="6763" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchURL -->
        <test-method signature="orderHistory()[pri:0, instance:parent.testClass.EndToEndTest@790da477]" started-at="2022-10-02T21:56:07 IST" name="orderHistory" finished-at="2022-10-02T21:56:08 IST" depends-on-methods="parent.testClass.EndToEndTest.EndToEndtest" duration-ms="1479" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- orderHistory -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:parent.testClass.EndToEndTest@790da477]" started-at="2022-10-02T21:56:08 IST" name="tearDown" finished-at="2022-10-02T21:56:08 IST" duration-ms="304" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- parent.testClass.EndToEndTest -->
    </test> <!-- EtoETest -->
    <test started-at="2022-10-02T21:56:09 IST" name="ErrorCheck" finished-at="2022-10-02T21:56:27 IST" duration-ms="17807">
      <class name="parent.testClass.EndToEndTest2ErrorCheck">
        <test-method is-config="true" signature="launchURL()[pri:0, instance:parent.testClass.EndToEndTest2ErrorCheck@f58853c]" started-at="2022-10-02T21:56:09 IST" name="launchURL" finished-at="2022-10-02T21:56:17 IST" duration-ms="7561" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchURL -->
        <test-method signature="EndToEndtest()[pri:0, instance:parent.testClass.EndToEndTest2ErrorCheck@f58853c]" started-at="2022-10-02T21:56:17 IST" name="EndToEndtest" finished-at="2022-10-02T21:56:17 IST" duration-ms="712" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [ncorrect email or password.] but found [Incorrect email or password.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [ncorrect email or password.] but found [Incorrect email or password.]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at parent.testClass.EndToEndTest2ErrorCheck.EndToEndtest(EndToEndTest2ErrorCheck.java:17)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- EndToEndtest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:parent.testClass.EndToEndTest2ErrorCheck@f58853c]" started-at="2022-10-02T21:56:18 IST" name="tearDown" finished-at="2022-10-02T21:56:18 IST" duration-ms="298" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="launchURL()[pri:0, instance:parent.testClass.EndToEndTest2ErrorCheck@f58853c]" started-at="2022-10-02T21:56:18 IST" name="launchURL" finished-at="2022-10-02T21:56:26 IST" duration-ms="7221" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchURL -->
        <test-method signature="EndToEndtestt()[pri:0, instance:parent.testClass.EndToEndTest2ErrorCheck@f58853c]" started-at="2022-10-02T21:56:26 IST" name="EndToEndtestt" finished-at="2022-10-02T21:56:27 IST" duration-ms="1005" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- EndToEndtestt -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:parent.testClass.EndToEndTest2ErrorCheck@f58853c]" started-at="2022-10-02T21:56:27 IST" name="tearDown" finished-at="2022-10-02T21:56:27 IST" duration-ms="254" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- parent.testClass.EndToEndTest2ErrorCheck -->
    </test> <!-- ErrorCheck -->
  </suite> <!-- Suite -->
</testng-results>
